Makefile

```
@@ -56,6 +78,68 @@ lint: ## ESLint, both stacks
 lint-md: ## markdownlint across the docs we maintain (see .markdownlint-cli2.jsonc)
 	npm run lint:md
 
+# Each external linter FAILS with an install hint rather than skipping. A gate
+# that quietly passes when its tool is absent is worse than no gate — that is
+# exactly the failure `lint-md` had before it reached .PHONY.
+
+# File lists come from $(LINT_FILES); see its definition for the scope rules.
+lint-sh: ## shellcheck every tracked shell script
```

checkmake (0.3.2)

[warning] Target body for "lint-sh" exceeds allowed length of 5 lines (9).

(maxbodylength)